### DISCUSSION TITLE
Clicking 'New View' always duplicates and replicates the original View

### DISCUSSION THREAD

**Discussion by ian-steffy** [community/community#12972] [Projects and Issues]
Hey everyone,

I have one View in Github Projects Beta and I would like to create a brand new one, however when I click 'New View' in the top tab, it just duplicates that original view and mimics all changes that original view makes. 

This seems like a bug. 

I want a fresh view that has its own behavior. A view that syncs with another view doesn't seem like a useful feature either.

**Comment by eclime** [comment]
I second this. I am having the same problem. Will this ever be fixed?

**Reply by abhinax4991** [reply]
facing same issue

**Comment by williamdarkocode** [comment]
any updates on this? makes no sense for both "New view" and "Duplicate View" to have the same function. Kinda stupid

**Comment by carter-0** [comment]
still broken, just deleted all my TODOs accidentally..
